# HG changeset patch
# User neugens
# Date 1507735508 -3600
#      Wed Oct 11 16:25:08 2017 +0100
# Node ID 5b91bf11d0f41a6124ffb0fcd8870507b50bd734
# Parent  7b1a2f967cd8e950fe7dc47d716af13d36239a38
8188030, PR3459, RH1484079: AWT java apps fail to start when some minimal fonts are present
Summary: Handle CFF fonts
Reviewed-by: andrew, prr

diff --git a/src/solaris/native/sun/awt/fontpath.c b/src/solaris/native/sun/awt/fontpath.c
--- openjdk/jdk/src/solaris/native/sun/awt/fontpath.c
+++ openjdk/jdk/src/solaris/native/sun/awt/fontpath.c
@@ -1233,6 +1233,7 @@
                 && (strcmp((char*)fontformat, "TrueType") != 0)
 #if defined(__linux__) || defined(_AIX)
                 && (strcmp((char*)fontformat, "Type 1") != 0)
+                && (strcmp((char*)fontformat, "CFF") != 0)
 #endif
              ) {
                 continue;
